app/models/api/access_token.rb

```diff
@@ -26,5 +26,15 @@ def initialize(value: nil, errors: nil)
       @value = value
       @errors = errors
     end
+
+    def expired?
+      begin
+        JWT.decode value, Rails.application.credentials.app.secret_access_key, true,
+                   { algorithm: 'HS256' }
+      rescue JWT::DecodeError
+        return true
+      end
+      false
+    end
```

Comment:
末尾の false にたどり着くフローが分かりづらいので、こう書くのはどうでしょうか

```suggestion
    def expired?
      JWT.decode value, Rails.application.credentials.app.secret_access_key, true,
                 { algorithm: 'HS256' }
      false
    rescue JWT::DecodeError
      true
    end
```

もしくはこうかな (書いてみたけど、上の方がおすすめだけど)

```suggestion
    def expired?
      begin
        JWT.decode value, Rails.application.credentials.app.secret_access_key, true,
                   { algorithm: 'HS256' }
      rescue JWT::DecodeError
        true
      else
        false
      end
    end
```